squad/frontend/templates/squad/project.jinja2

```diff
@@ -25,3 +25,7 @@
 
 {% endwith %}
 {% endblock %}
+
+{% block javascript %}
+<script type="module" src='{{static("squad/project.js")}}'></script>
```

Comment:
This file doesn't exist. Did you mean to include in this PR?

Reply:
ah, that's leftover from AJAX implementation that I abandoned. Will remove.